### ISSUE TITLE
MAC USERS PLEASE READ BEFORE SUBMITING ISSUE

### ISSUE THREAD

**Issue by Anjok07** [Anjok07/ultimatevocalremovergui#367]
If you downloaded the latest patch and found that it wasn't compatible or could not open with your MAC, please download the x86_64 version of the application. See the instructions here - [MacOS Installation](https://github.com/Anjok07/ultimatevocalremovergui#macos-installation)

**Comment by codplay89** [comment]
when you say latest patch, do you mean just running the dmg file to either the m1 downloader or intel downloader? or is there a separate patch file to install along side the dmg file?

**Comment by Anjok07** [comment]
> when you say latest patch, do you mean just running the dmg file to either the m1 downloader or intel downloader? or is there a separate patch file to install along side the dmg file?

The patch is not separate in this case. Since you have an M1, you would need to download the M1 installer, or you can just download the file straight from the GUI (they're both the same file), then replace your current installation with the one in the new DMG.

**Comment by codplay89** [comment]
dude thank you so much man. I appreciate all of your help and have been absolutely loving uvr 5. Amazing work

**Comment by Anjok07** [comment]
> dude thank you so much man. I appreciate all of your help and have been absolutely loving uvr 5. Amazing work

I'm very happy to hear that! Please let me know if you run into any issues.

**Comment by codplay89** [comment]
So far, everything is running amazing. The speed in which a song gets exported in ensemble is faster than the speeds on Mvsep.com. Because Im new to this whole coding and instrumental making thing, I didn't know too much about how to use the program, but with a little help on here in the forums, people have pointed me in the right direction and I have concurred that uvr 5 is miles ahead of any major vocal removal software EVER. And I have used pretty much all of them. Can't thank you enough for all your work. 

Will you guys keep making uvr's? or do you think you'll stop making them at some point?

**Comment by Anjok07** [comment]
Yes, UVR will continue to be maintained, and new models and features will be added over time. 

Thank you for the support!
